cocotb/decorators.py

```diff
@@ -324,8 +324,9 @@ def __str__(self):
 class function(object):
     """Decorator class that allows a function to block.
 
-    This allows a function to internally block while
-    externally appear to yield.
+    This allows a coroutine that consumes simulation time 
+    to be called by a normal thread; in other words, 
```

Comment:
```suggestion
    to be called by a thread started by :func:`external`; in other words, 
```

Reply:
Done. Those wording changes came from ``hal_cosimulation.rst`` by the way, maybe I should do the same changes there...